runtime/POSIX/klee_init_env.c

```diff
@@ -116,6 +118,14 @@ usage: (klee_init_env) [options] [program arguments]\n\
   -sym-stdin <N>            - Make stdin symbolic with size N.\n\
   -sym-stdout               - Make stdout symbolic.\n\
   -sym-pmem <FILE> <N>      - Provide a symbolic persistent memory file and size.\n\
+  -sym-pmem-init-concrete <INIT_FILE_PATH/0>\n\
+                            - Initialize the persistent memory file to a \n\
+                              concrete value, either based on values from \n\
+                              a real file or to all zeros if provided.\n\
```

Comment:
> based on values from a real file

ooh, nice, could be useful

Reply:
It is! That's how I found the other round of bugs.